pkgs/applications/misc/rlaunch/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkgs
```

please inherit them from `xorg` inside `all-packages.nix`
```suggestion
, libX11
, libXft
, libXinerama
```

---

That is optional but pkgs needs to be dropped.